.storybook/preview.js

```diff
@@ -9,3 +9,5 @@ Vue.use(RipeComponentsVue);
 Vue.component("global-events", GlobalEvents);
 
 Vue.prototype.$bus = new Vue();
+
+export const parameters = { layout: "fullscreen" };

```

Comment:
This was added to fix the storybook not being in fullscreen, thus always showing an horizontal scroll bar.

Reply:
Please move this to a separate PR. I know it may sound like too much trouble, but we really need to keep it 1 PR = 1 concern.

This makes merging faster, so it benefits everyone.